### ISSUE TITLE
Image on user personas page isn't accessible

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [ONSdigital/sdg-indicators#3613]
**URL:** http://uk-sdg-feature-branches.s3-website.eu-west-2.amazonaws.com/feature-dac-testing/user-personas/
**Page title:** User personas and their needs -U.K. Indicators For The Sustainable Development Goals

**Screenshot:**

![image](https://user-images.githubusercontent.com/43404358/188176514-80515de4-ff68-427a-a15b-800b6f13b445.png)

**Issue:**
When the page is magnified, line spacing is reduced and text overlaps so that it is difficult to read. 

**Solution:**
Ensure that the page can be viewed as a single column at 320px without any loss of content or functionality.For more information, please refer to F102: Failure of Success Criterion 1.4.10 due to content disappearing and not being available when content has reflowed.Please note that for content which requires atwo-dimensional layout for usage or meaningincluding tabular information, horizontal scrolling may be necessary in order for the presentation and structure to be preserved.

Additionally arrow doesn't change colour in high contrast mode

**Comment by LucyGwilliamAdmin** [comment]
@brockfanning will you have chance to look at this one soon please so it's ready for the retest?

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin I just put up a PR that is intended to use a "flex" approach for a better zoom behavior.
